src/style.css

```
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  outline: none;
+}
+
+body {
+  margin: 20px 8px;
+  font-weight: 300;
+}
+
+.search {
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.search object {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.searchbar {
+  width: 100%;
+  font-family: Avenir, sans-serif;
```

move to `html` tag and use inherit here
```suggestion
  font-family: inherit;
```